src/Silverback.Integration.Kafka/Messaging/Messages/KafkaStatisticsEvent.cs

```diff
@@ -31,6 +32,17 @@ public KafkaStatisticsEvent(string statistics)
         /// <summary>
         ///     Gets the statistics as typed <see cref="KafkaStatistics"/> object
         /// </summary>
-        public KafkaStatistics Statistics { get; }
+        public KafkaStatistics Statistics
```

Comment:
Come on, this could have been a one-liner in c#:

```c#
public KafkaStatistics Statistics => _statistics ??= JsonConvert.DeserializeObject<KafkaStatistics>(Raw);
```
:wink:

Reply:
I know, but I think the if-statment is more readable...